cmd/gendocs/BUILD

```diff
@@ -9,6 +9,7 @@ load(
 go_binary(
     name = "gendocs",
     embed = [":go_default_library"],
+    pure = "off",
```

Comment:
Can we put the pure = off commit in its own PR?

Reply:
this has been replaced with more complicated platform-specific logic, so it probably should remain a part of this PR.